README.md

```diff
@@ -2,727 +2,369 @@
 [![PyPI version](https://badge.fury.io/py/mbed-ls.svg)](https://badge.fury.io/py/mbed-ls)
 
-# Table of contents
-
-* [Description](#description)
-* [Rationale](#rationale)
-* [Installation](#installation)
-  * [Installation from PyPI (Python Package Index)](#installation-from-pypi-python-package-index)
-  * [Installation from Python sources](#installation-from-python-sources)
-* [mbedls as command line tool](#mbedls-as-command-line-tool)
-  * [Exporting mbedls output to JSON](#exporting-mbedls-output-to-json)
-* [Porting instructions](#porting-instructions)
-  * [mbed-enabled technical specification overview](#mbed-enabled-technical-specification-overview)
-    * [TargetID as device unique identifier](#targetid-as-device-unique-identifier)
-  * [mbed-ls auto-detection approach for Ubuntu](#mbed-ls-auto-detection-approach-for-ubuntu)
-* [Retarget mbed-ls autodetection results](#retarget-mbed-ls-autodetection-results)
-  * [mbedls.json file properties](#mbedlsjson-file-properties)
-  * [Example of retargeting](#example-of-retargeting)
-* [Mocking new or existing target to custom platform name](#mocking-new-or-existing-target-to-custom-platform-name)
-  * [Mock command line examples](#mock-command-line-examples)
-  * [Mocking example with Freescale K64F platform](#mocking-example-with-freescale-k64f-platform)
-* [mbed-ls unit testing](#mbed-ls-unit-testing)
-  * [Code coverage](#code-coverage)
-* [Configure mbed-enabled device to work with your host](#configure-mbed-enabled-device-to-work-with-your-host)
-  * [Windows serial port configuration](#windows-serial-port-configuration)
-  * [Mounting with sync](#mounting-with-sync)
-    * [Ubuntu](#ubuntu)
-  * [Raspberry Pi - Raspbian Jessie Lite](#raspberry-pi---raspbian-jessie-lite)
-    * [Prerequisites](#prerequisites)
-    * [Install LDM](#install-ldm)
-    * [Enable LDM](#enable-ldm)
-    * [Making sure LDM is active (running)](#making-sure-ldm-is-active-running)
-* [Known issues](#known-issues)
-
-# Description
-
-```mbed-ls``` is a Python (2 and 3) module that detects and lists mbed-enabled devices connected to the host computer. It will be delivered as a redistributable Python module (package) and command line tool.
-
-Currently supported operating system:
-
-* Windows 7.
-* Linux.
-* Mac OS X (Darwin).
-* Raspbian Jessie Lite.
-
-# Rationale
-
-When connecting more than one mbed-enabled device to the host computer, it takes time to manually check the platforms' binds:
-
-* Mount point (MSD / disk).
-* Virtual serial port (CDC).
-* mbed's TargetID and generic platform name.
-
-```mbed-ls``` provides these points of information for all connected boards at once in a simple console (terminal) output.
+# mbed-ls
+
+`mbed-ls` is a Python (2 and 3) module that detects and lists Mbed Enabled devices connected to the host computer. It is delivered as a redistributable Python module (package) and command-line tool. It works on all major operating systems (Windows, Linux and Mac OS X).
```

Comment:
Query: Who or what delivers this as a redistributable Python module (package) and command-line tool? Mbed OS? The Mbed team? Something else? Please clarify for active voice.

Reply:
Could we change this to: Mbed LS is published by the Mbed OS team on PyPI